rushstack-internal-rigs/heft-rushstack-internal-node-rig/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "heft-rushstack-internal-node-rig",
+  "version": "0.1.0",
```

Comment:
Unpublished projects don't get version bumps. The version should be something like 0.0.0 or 1.0.0 right?